README.md

```diff
@@ -46,6 +46,7 @@ func mapper(event couchbase.Event) []message.KafkaMessage {
 			Headers: nil,
 			Key:     event.Key,
 			Value:   event.Value,
+			Writer:  event.Value
```

Comment:
Possible typo here, WriterData :D we can add comment like

WriterData: event.Value // ıf you want to use completion callback, it logged bla bla